components/css/partials/_highlight.scss

```diff
@@ -0,0 +1,31 @@
+.highlight {
+    margin: 1em 0;
+    padding-left: 2em;
```

Comment:
Perhaps we should consider reducing this padding on smaller devices to save a little bit of horizontal space.